### PR TITLE
fix(table): added tr border to sticky nested table header

### LINKED ISSUE #4846: Bug: Nested table header breaks with sticky header
When scrolling starts in the table the headers that do not have nested subheadings appear broken out above the normal heading.
Screen Shot 2022-03-07 at 12 41 51 PM

An example was prepared and can be found here https://codesandbox.io/s/vibrant-goldstine-1t2b68?file=/index.tsx

The nested table header is used on the OpenShift Network Observability operator. This is in Tech preview now and scheduled for full release with 4.11.

### PR DESCRIPTION
**Pull request by mattnolting**
closes #4846 

https://patternfly-pr-4857.surge.sh/components/table#nested-column-headers
https://patternfly-pr-4857.surge.sh/components/table#nested-column-headers-sticky-header

### CODE REVIEW

**Comment by kmcfaul** [comment]
It looks like in the preview the new example is rendering the hbs as text instead of visually.

I've got a react update in place locally for when this PR goes in as well.

File: src/patternfly/components/Table/table-tr.hbs [hunk 1]
```diff
@@ -26,14 +27,20 @@
...
+  {{/unless}}
   {{#if table-tr--IsHoverable}}
     tabindex="0"
   {{/if}}
   {{#if table-tr--attribute}}
     {{{table-tr--attribute}}}
   {{/if}}>
-  {{> @partial-block}}
+  {{#if table-tr--IsBorderRow}}
+    {{> table-td table-td--IsEmpty="true" table-td--attribute='colspan="100%" aria-hidden="true" tabindex="-1"'}}
```
Comment (mcoker) [inline src/patternfly/components/Table/table-tr.hbs:42]: > `colspan="100%"`

Is that valid? From MDN, looks like this needs to be an integer https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attributes

File: src/patternfly/components/Table/examples/Table.md [hunk 2]
```diff
@@ -3473,67 +3473,79 @@ For sticky columns to function correctly, the parent table's width must be contr
...
+      {{> table-tr--nested-column-header table-tr--index="10"}}
+    {{/table-tbody}}
+  {{/table}}
+</div>
+```
+
+### Nested column headers, sticky header
+```hbs
+<div class="pf-c-scroll-inner-wrapper">
+  {{#> table table--id="table-nested-column-headers-sticky-header-example" table--modifier="pf-m-grid-md pf-m-sticky-header" table--grid="true" table--attribute='aria-label="This is a nested column header table example"'}}
```
Comment (mcoker) [inline src/patternfly/components/Table/examples/Table.md:3495]: nit - 

```suggestion
  {{#> table table--id="table-nested-column-headers-sticky-header-example" table--modifier="pf-m-grid-md pf-m-sticky-header" table--grid="true" table--attribute='aria-label="This is a sticky nested column header table example"'}}
```

**Review by mcoker — COMMENTED** [review]
LGTM! 2 small comments.

**Review by mcoker — APPROVED** [review]
L🍕TM!

**Pull request merged**